packages/protocol/test/Vault.t.sol

```diff
@@ -30,6 +37,10 @@ contract VaultTest is DSTestPlus {
   uint256 bobPkey = 0xB;
   address bob = vm.addr(bobPkey);
 
+  // These test prices should be inverse of each other.
+  uint256 public constant TEST_USD_PER_ETH_PRICE = 2000e18;
+  uint256 public constant TEST_ETH_PER_USD_PRICE = 5e14;
```

Comment:
it's a good idea to set these values as constants but why prefixing them with `TEST_`?